torchgeo/datasets/ssl4eo_downstream_landsat.py

```diff
@@ -0,0 +1,195 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Self-Supervised Learning for Earth Observation Downstream Evaluation."""
+
+import glob
+import os
+from typing import Callable, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import rasterio
+import torch
+from torch import Tensor
+
+from .geo import NonGeoDataset
+from .utils import check_integrity, extract_archive
+
+
+class SSL4EODownstream(NonGeoDataset):
```

Comment:
Trying to decide if we want 1 class (SSL4EO-L CDL/NLCD) or 2 (SSL4EO-L CDL and SSL4EO-L NLCD).

Reply:
We should at least include L in the dataset name since it is only for Landsat. I'm hoping we don't have to do this kind of thing again in the future since other satellites have more downstream datasets already available.